src/server/command_handler.py

```diff
@@ -17,17 +17,44 @@
 
 from src.utils.logging_manager import LoggingManager
 
+from src.parser.parser import Parser
+from src.optimizer.statement_to_opr_convertor import StatementToPlanConvertor
+
+from src.executor.plan_executor import PlanExecutor
+from src.optimizer.plan_generator import PlanGenerator
+from src.utils.logging_manager import LoggingManager
+
 
 @asyncio.coroutine
-def handle_request(transport, request_message):
+def handle_request(transport, query):
     """
         Reads a request from a client and processes it
 
         If user inputs 'quit' stops the event loop
         otherwise just echoes user input
     """
+    parser = Parser()
+    eva_statement = parser.parse(query)
+    insert_statement = eva_statement[0]
+
+    LoggingManager().log("Result from the parser: " + str(insert_statement))
+
+    convertor = StatementToPlanConvertor()
+    convertor.visit(insert_statement)
+
+    convertor.plan
+
+    eva_statement = parser.parse(query)
+    response_message = ""
+
+    for i in range(len(eva_statement)):
```

Comment:
Any reason for not iterating over the statements directly?

for stmt in eva_statement: ....

Also, can we rename, eva_statement to eva_statements -> as it's a list of statements?

Reply:
I agree. We should rewrite this.